src/ts/forwarders/forwarder.ts

```diff
@@ -116,34 +103,23 @@ export abstract class Forwarder<TParams extends Partial<MinimalTx>> {
    * @param tx ProxyAccountCallData or RelayCallData
    */
   public async signAndEncodeMetaTransaction(tx: TParams): Promise<MinimalTx> {
-    if (tx.to) {
-      // we can cast here since we know to is defined
-      const forwardParams = await this.signMetaTransaction(
-        tx as RequiredTo<TParams>
-      );
-      const encodedData = await this.encodeSignedMetaTransaction(forwardParams);
-      return { to: forwardParams.to, data: encodedData };
-    } else if (tx.data) {
-      const deploymentParams = await this.signMetaDeployment(tx.data);
-      const encodedData = await this.encodeSignedMetaDeployment(
-        deploymentParams
-      );
-      return { to: deploymentParams.to, data: encodedData };
-    } else throw new Error("Either 'to' or 'data' must be supplied.");
+    const forwardParams = await this.signMetaTransaction(
+      tx as RequiredTarget<TParams>
```

Comment:
https://github.com/anydotcrypto/metatransactions/blob/master/src/ts/forwarders/forwarder.ts#L121 its currently in master. i dunno what to do with it?

Reply:
So the reason it was safe before was because on the line above we checked that tx.to existed -we're not doing that anymore so it could throw an error. I'll have a go at the types and try to commit something